docker-entrypoint.sh

```diff
@@ -27,7 +23,7 @@ disown
 
 echo --- test-server ----
 cd /server/cmd/testing-server/
-echo Building test-server....
-go build
-echo Running test server...
-./testing-server
\ No newline at end of file
+echo Building and running test-server....
+# go get github.com/markbates/refresh
```

Comment:
do this need to be un commented?

Reply:
No.  I think it will work, as it is a requirement for the project.  I just have a wee problem whish is that I need to rerun it because the mac version doesn't work in linux and vice versa.

Reply:
`Building and running test-server....
/docker-entrypoint.sh: line 34: refresh: command not found
/docker-entrypoint.sh: line 35: refresh: command not found`
for me it needs to be uncommented, also as soon as you run `go mod tidy` it will remove the references as none of the cmd packages use it